salesforce/client.bal

```diff
@@ -95,42 +113,151 @@ public isolated client class Client {
+    @display {label: "Get Available API Versions"}
+    @deprecated
+    isolated remote function getAvailableApiVersions() returns @display {label: "Versions"} Version[]|Error {
+        string path = utils:prepareUrl([BASE_PATH]);
+        json res = check self.get(path);
+        return toVersions(res);
+    }
+
+    # Lists the resources available for the specified API version.
+    #
+    # + apiVersion - API version (v37)
+    # + return - `Resources` as map of strings if successful. Else, the occurred `Error`.
+    # 
+    # # Deprecated
+    # This function is deprecated as the `salesforce` client is deprecated.
+    # Use the `getResourcesByApiVersion` function in `salesforce.rest` client instead.
+    @display {label: "Get Resources by API Version"}
+    @deprecated
+    isolated remote function getResourcesByApiVersion(@display {label: "API Version"} string apiVersion) 
+                                                      returns @display {label: "Resources"} map<string>|Error {
+        string path = utils:prepareUrl([BASE_PATH, apiVersion]);
+        json res = check self.get(path);
+        return toMapOfStrings(res);
+    }
+
+    # Lists the Limits information for your organization.
+    #
+    # + return - `OrganizationLimits` as map of `Limit` if successful. Else, the occurred `Error`.
+    # 
+    # # Deprecated
+    # This function is deprecated as the `salesforce` client is deprecated.
+    # Use the `getOrganizationLimits` function in `salesforce.rest` client instead.
+    @display {label: "Get Organization Limits"}
+    @deprecated
+    isolated remote function getOrganizationLimits() 
+                                                   returns @display {label: "Organization Limits"} 
+                                                   map<Limit>|Error {
+        string path = utils:prepareUrl([API_BASE_PATH, LIMITS]);
+        json res = check self.get(path);
+        return toMapOfLimits(res);
+    }
+
+
+    # Accesses records based on the specified object ID, can be used with external objects.
+    # 
+    # + path - Resource path
+    # + return - JSON result if successful else or else `sfdc:Error`
+    # 
+    # # Deprecated
+    # This function is deprecated as it returns a json response.
+    # Use the new and improved `getById(string sObject, string id)` function instead.
+    @display {label: "Get Record"}
+    @deprecated
+    isolated remote function getRecord(@display {label: "Resource Path"} string path) 
+                                       returns @display {label: "Result"} json|Error {
+        json|http:ClientError response = self.salesforceClient->get(path);
+        if response is json {
+            return response;
+        } else {
+            return checkAndSetErrorDetail(response);
+        }
+    }
+
+    private isolated function get(@display {label: "Resource Path"} string path) 
+                         returns @display {label: "Result"} json|Error {
```

Comment:
Fix the formatting issue here

Reply:
Fixed in 8bfb122ccf270a0868c40a7ea6ddaef1d3a79a39